<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1441</width>
    <height>924</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>551</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="EntryFilesLayout">
     <item row="4" column="0">
      <widget class="QCheckBox" name="DeleteAfterCheckBox">
       <property name="text">
        <string>–£–¥–∞–ª—è—Ç—å –ø–æ—Å–ª–µ –æ–±—Ä–∞–±–æ—Ç–∫–∏</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="EnrtyDirLayout">
       <item>
        <widget class="QLabel" name="EntDirlabel">
         <property name="text">
          <string>[üìÅ] –í—Ö–æ–¥–Ω–∞—è –ø–∞–ø–∫–∞</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="EntryDirLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="InputDirButton">
         <property name="text">
          <string>–û–±–∑–æ—Ä ...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>–ú–∞—Å–∫–∞</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>---–í—Ö–æ–¥–Ω—ã–µ —Ñ–∞–π–ª—ã---</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>551</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="OutpurDirLabel">
         <property name="text">
          <string>[üìÅ] –í—ã—Ö–æ–¥–Ω–∞—è –ø–∞–ø–∫–∞</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="OutputDirLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="OutputDirButton">
         <property name="text">
          <string>–û–±–∑–æ—Ä ...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="OutputFilesLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>---–í—ã—Ö–æ–¥–Ω—ã–µ —Ñ–∞–π–ª—ã---</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>–ü–æ–≤—Ç–æ—Ä—è—é—â–∏–µ—Å—è –∏–º–µ–Ω–∞</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="NamesCollisionComboBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>440</y>
      <width>551</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>XOR –∑–Ω–∞—á–µ–Ω–∏–µ (16 hex):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="XORLineEdit"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>–†–µ–∂–∏–º</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="WorkModeComboBox"/>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>---–ü–∞—Ä–∞–º–µ—Ç—Ä—ã---</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>–ò–Ω—Ç–µ—Ä–≤–∞–ª (—Å–µ–∫)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="IntervalSpinBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>660</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="StartButton">
       <property name="text">
        <string>‚ñ∂Ô∏è –°—Ç–∞—Ä—Ç</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="StopButton">
       <property name="text">
        <string>‚èπÔ∏è –°—Ç–æ–ø</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>510</y>
      <width>551</width>
      <height>190</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>–ü—Ä–æ–≥—Ä–µ—Å—Å</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>–°—Ç–∞—Ç—É—Å:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="StatusLineEdit"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>–õ–æ–≥</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="LogEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1441</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
